hello-world/README.md

```diff
@@ -30,7 +30,7 @@ This repo has a few significant directories:
 
 ### 2. Install dependencies
 
-Open your terminal and change into the Hello World repo root folder (`cms-js-building-blocks-hello-world/`) if you are not already there.
+Open your terminal and change into the Hello World repo root folder (`cms-js-building-blocks-hello-world/hello-world`) if you are not already there.
```

Comment:
the root repo name got updated as well since we wrote this

```suggestion
Open your terminal and change into the Hello World repo root folder (`cms-js-building-block-examples/hello-world`) if you are not already there.
```